.classpath

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
bu dosya ismini .gitignore dosyasına ekle. PRdan da sil. Bunun repoda olmaması gerekiyor

Reply:
Bu işlemi tamamladım. (Yorumlara cevap verip resolve etmek için bu şekilde yazacağım hocam.)